packages/pluggableWidgets/image-viewer-web/src/components/Lightbox.tsx

```diff
@@ -0,0 +1,57 @@
+import { useOnClickOutside } from "@mendix/piw-utils-internal";
+import classNames from "classnames";
+import { createElement, ReactElement, useRef } from "react";
+import ReactModal from "react-modal";
+
+import closeIconSvg from "../assets/ic24-close.svg";
+
+interface LightboxProps {
+    children: ReactElement;
+    isOpen: boolean;
+    onClose: () => void;
+}
+
+const modalStyle = {
+    // Only way to style the content div of `ReactModal` is through inline styles.
```

Comment:
Does this means customers cannot customize using Atlas?

Reply:
I will have to dive deeper into this one

Reply:
👍🏻